cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -91,11 +91,17 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     protected static final String TLS_CC_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cc-certs/";
     protected static final String TLS_CA_CERTS_VOLUME_NAME = "cluster-ca-certs";
     protected static final String TLS_CA_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cluster-ca-certs/";
+    protected static final String CRUISE_CONTROL_CONFIG_VOLUME_NAME = "config";
+    protected static final String CRUISE_CONTROL_SERVER_CONFIG_VOLUME_NAME = "server-config";
+    protected static final String CRUISE_CONTROL_CAPACITY_CONFIG_VOLUME_NAME = "capacity-config";
```

Comment:
Why do we need 3 different volume names for one config? Are these actually volume names and not just file names?

Reply:
Fix the variable names, yes they are just filenames